src/Microsoft.Health.Fhir.Api/Configs/NetworkConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Api.Configs;
+
+public class NetworkConfiguration
```

Comment:
Can we add a comment on this, doesn't seem used?

Reply:
Comments added! yes will be mapping the values in wp